src/sqlancer/GlobalState.java

```diff
@@ -118,6 +125,54 @@ public boolean executeStatement(Query q) throws SQLException {
         return success;
     }
 
+    public ResultSet executeStatementAndGet(Query q) throws SQLException {
+        boolean logExecutionTime = getOptions().logExecutionTime();
+        ExecutionTimer timer = logExecution(q, logExecutionTime);
+        ResultSet result = manager.executeAndGet(q);
+        if (result != null && getOptions().printSucceedingStatements()) {
+            System.out.println(q.getQueryString());
+        }
+        if (logExecutionTime) {
+            getLogger().writeCurrent(" -- " + timer.end().asString());
+        }
+        if (q.couldAffectSchema()) {
+            updateSchema();
+        }
+        return result;
+    }
+
+    public boolean fillAndExecuteStatement(Query q, String template, List<String> fills) throws SQLException {
+        boolean logExecutionTime = getOptions().logExecutionTime();
+        ExecutionTimer timer = logExecution(q, logExecutionTime);
+        boolean success = manager.fillAndExecute(q, template, fills);
+        if (success && getOptions().printSucceedingStatements()) {
```

Comment:
Could we add a method for the common code that is executed after a statement or query is executed? We could name it `executeQueryEpilogue` or so.